.circleci/config.yml

```diff
@@ -161,18 +164,31 @@ jobs:
       - *import_image
       - run: |
           mkdir ~/testresults
-          docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v ~/testresults:/output signalfx-agent-dev pytest -n8 --junitxml=/output/integration_tests.xml -m "not packaging and not installer and not k8s" tests
+          docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v ~/testresults:/output signalfx-agent-dev \
```

Comment:
Is it possible to run these outside of the dev image like your k8s tests do?

Reply:
I'll take a look. Other than pytest and the agent binary, are there any other known dependencies from the dev image required for the tests?

Reply:
The collectd bundle, but you can get that from the final agent image by dumping the contents of it to the local filesystem on Circle.